### DISCUSSION TITLE
LEDのショートについて

### DISCUSSION THREAD

**Discussion by Kazuki1320** [Yowkees/keyball#226] [Q&A]
Keyball44のボール側PCB（右手側）を組み立てましたところ、
LEDの10番までは正常に点灯し、11番から先が点灯しませんでした。

はんだも問題なさそうでしたので、No11を熱でLEDを壊したかと思い別のLEDを付け直しても治りませんでした。

そこで導通チェッカーでみたところ、No10の DOUT とNo11のDINは繋がっていたのですが、
No11のDINとNo11のVCCがショートしているようです。

どのように対応すればいいか教えていただければ幸いです。
お手数ですがご確認お願いいたします。

![IMG_1408のコピー](https://user-images.githubusercontent.com/132678804/236475814-b91b3dbc-cecb-4fb3-96be-efb7b7dc86b7.jpeg)

追記
LEDを外して導通チェッカーで見たところやはりNo11のDINとNo11のVCCがショートしていました。
![IMG_1420](https://github.com/Yowkees/keyball/assets/132678804/27cf9365-e186-4242-b701-345df217ab8e)

**Comment by Kazuki1320** [comment]
修理が難しそうでしたら、ボール側のPCB、ダイオード、LEDなどを販売していただけるとありがたいです。
人気のため在庫がないと思いますが、ご確認お願いいたします。

**Comment by Yowkees** [comment]
ご連絡が遅くなり申し訳ありませんでした。

とてもキレイに半田付けできていますので点灯しそうに見えますが、何か半田カスみたいなのが付着しましたでしょうか。

後は、No10のLEDを一度外していただくとDINの線が完全に孤立しますので、それでもVCCとショートしているようでしたらどこでショートしているか探ることになるかと思います。
もちろん基板単体の販売も可能です。直らないようでしたら配送先等ご相談したいので、下記の白銀ラボウェブショップのお問い合わせよりご一報いただければ幸いです。
https://shirogane-lab.com/pages/contact

以上、お手数ですがご検討お願いいたします。

**Comment by Kazuki1320** [comment]
回答ありがとうございます。
No.10のLEDを外し、再度はんだ付けしましたところ、無事全てのLEDが点灯いたしました。
お手数おかけいたしました。

とても素晴らしいキーボードです。
大切に使わせていただきます。

**Comment by Yowkees** [comment]
無事に直ったようで良かったです！安心しました。
ぜひ末永くご愛用いただければ幸いです。
